src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,69 @@
+.watch {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+
+  &__page {
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+    @include watch-size;
+
+    border-radius: 50%;
+    @include background-color;
+  }
+
+  &__minute {
+    position: $absolut-position;
+    top: 20px;
+
+    height: 180px;
+    width: 2px;
+
+    border: 2px solid transparent;
+    @include arrow-gradient;
+    border-radius: 3px;
+
+    transform-origin: bottom;
+    animation: $animation-name 3600s $time-duration linear;
+  }
+
+  &__second {
+    position: $absolut-position;
+    top: 70px;
+
+    height: 130px;
+    width: 2px;
+
+    border: 2px solid transparent;
+    @include arrow-seconds-color;
+    border-radius: 3px;
+
+    transform-origin: bottom;
+    animation: $animation-name 60s $time-duration linear;
+  }
+
+  &__middle {
+    position: $absolut-position;
+    top: 198px;
```

Comment:
What will be when you will decide to change the clock size?
Can we calculate the position of this element somehow?
The same for arrows